tests/particle_output_multiple_formats/statistics

```diff
@@ -0,0 +1,16 @@
+# 1: Time step number
+# 2: Time (seconds)
+# 3: Number of mesh cells
+# 4: Number of Stokes degrees of freedom
+# 5: Number of temperature degrees of freedom
+# 6: Number of degrees of freedom for all compositions
+# 7: Iterations for temperature solver
+# 8: Iterations for composition solver 1
+# 9: Iterations for Stokes solver
+# 10: Velocity iterations in Stokes preconditioner
+# 11: Schur complement iterations in Stokes preconditioner
+# 12: Time step size (seconds)
+# 13: Number of advected particles
+# 14: Particle file name(s)
```

Comment:
I think listing the full file name is appropriate, as it is right now. One could argue that it makes more sense to do this *relative to the statistics* file, though, which is itself in the `output-part....` directory. But that is a separate step, independent of the question here. I'm ok if we leave things as they are right now in this patch.